sphere-common/src/main/java/io/sphere/sdk/search/RangeTermModel.java

```diff
@@ -13,4 +13,19 @@
      * @return the model to specify the range of values from which to obtain the facets
      */
     RangedFacetSearchModel<T, V> faceted();
+
+    /**
+     * Gets an untyped search model (expecting simple Strings) while keeping the same search model path as this.
+     * This untyped search model allows you to build filters and facets, both range and term models.
+     * @return the untyped search model for this instance
+     */
+    default UntypedSearchModel<T> untyped() {
```

Comment:
If the `default` is not of your liking I could create 2 abstract classes that implement it (one for terms and one for ranges).